packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -787,29 +811,122 @@ test.serial('Settlement for unknown transaction (operator down)', async t => {
   const opDown = makeCustomer('Otto', cctp, txPub.publisher, feeConfig);
 
   const bridgePos = snapshot();
-  const sent = await opDown.sendFast(t, 20_000_000n, 'osmo12345');
+  const EUD = 'osmo10tt0';
+  const mintAmt = 5_000_000n;
+  const sent = await opDown.sendFast(t, mintAmt, EUD);
   await mint(sent);
   const bridgeTraffic = since(bridgePos);
 
   t.like(
     bridgeTraffic.bank,
     [
       {
-        amount: '20000000',
+        amount: String(mintAmt),
         sender: 'faucet',
         type: 'VBANK_GRAB',
       },
       {
-        amount: '20000000',
+        amount: String(mintAmt),
         recipient: 'agoric1qyqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc09z0g',
         type: 'VBANK_GIVE',
       },
     ],
     '20 USDC arrive at the settlement account',
   );
-  t.deepEqual(bridgeTraffic.local, [], 'no IBC transfers');
 
   await transmitTransferAck();
+  t.deepEqual(bridgeTraffic.local, [], 'no IBC transfers');
+
+  // activate oracles and submit evidence; expect Settler to forward (slow path)
+  // 'C12 - Contract MUST only pay back the Pool (fees) only if they started the advance before USDC is minted',
```

Comment:
This is maybe not _entirely_ relevant. We could adjust the test to fail (`AdvanceFailed`), and we would not pay fees to the pool here. This is covered in `settler.test.ts`, though.